src/generate/register.rs

```diff
@@ -261,16 +274,70 @@ pub fn render_register_mod(
                 config,
             )?;
         }
+    } else if register.read_action.is_some() {
+        if let Some(feature) = &config.impl_debug_feature {
+            r_debug_impl.extend(quote! {
+                #[cfg(feature=#feature)]
+            });
+        }
+        r_debug_impl.extend(quote! {
+            impl core::fmt::Debug for crate::generic::Reg<#regspec_ident> {
+                fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+                    write!(f, "(not read/has read action!)")
```

Comment:
```suggestion
                    write!(f, "(not readable)")
```

Reply:
I'd like it better if the register didn't include a debug impl if it never could be printed, but that would mean having to do a custom debug impl for blocks

Reply:
Yea - and thats the way I started off but then its not consistant for the developer who maybe tries to print a register contents and its different for write-only vs readable.  Main reason was making the logic work in peripherals.rs so that it exactly matched what is in registers.rs for registers/arrays/readable/read-action/write-only so I punted and just implemented Debug for all - it was much easier.

Reply:
I'm not sure about `(not readable)` as thats not correct, how about `(has read action)`

Reply:
or `has read side effects`

Reply:
maybe even just `(side effects)`?

Reply:
actually - what if we dropped one of the else if conditions and just printed "not readable" for write only or read action?

Reply:
im ok with any of these 👍🏼

Reply:
Done